metron-interface/metron-alerts/README.md

```diff
@@ -45,7 +45,7 @@ Alerts that are contained in a a meta alert are generally excluded from search r
 * The Management UI should be installed (which includes [Express](https://expressjs.com/))
 * The alerts can be populated using Full Dev or any other setup
 * UI is developed using angular4 and uses angular-cli
-* node.JS >= 7.8.0
+* nvm (or a similar node verison manager) should be installed. The node version required for this project is listed in the [.nvmrc](https://github.com/creationix/nvm#nvmrc) file.
```

Comment:
Do you have any recommendations for how we should install NVM?  I am assuming a `brew install nvm` will do the trick on a Mac?  Or do you prefer what is mentioned [here](https://github.com/creationix/nvm#install-script)?

Maybe we should embed a link to [this](https://github.com/creationix/nvm#install-script) if you think this is the right way to install.

Reply:
However we update the instructions here, we should also do the same in `metron-interface/metron-config/README.md`.

Reply:
We should also update the line above (line 47.): 

> UI is developed using angular4 and uses angular-cli

to: 

> UI is developed using angular 6 and uses angular-cli

Reply:
@ruffle1986 Fixed. I also updated `angular-cli` to `Angular CLI` since that seems to be how it's referenced [when mentioned in documentation](https://cli.angular.io/). Thanks for catching that!